src/aurora/dlgfile.cpp

```diff
@@ -226,6 +226,9 @@ void DLGFile::readEntry(const GFF3Struct &gff, Entry &entry) {
 
 	entry.line.sound = gff.getString("Sound");
 
+	if (gff.hasField("VO_ResRef"))
+		entry.line.voice = gff.getString("VO_ResRef");
```

Comment:
You don't need that `if` here. `GFF3Struct::getString()` returns its second parameter (defaulting to "") when the field doesn't exist.

You only need to check beforehand on `GFF3Struct::getStruct()` and `GFF3Struct::getList()`. These throw when no such field exists.